dev/package.json

```diff
@@ -14,27 +14,27 @@
         "dev": "vite"
     },
     "dependencies": {
-        "@khanacademy/kas": "^0.4.13",
-        "@khanacademy/kmath": "^0.3.2",
-        "@khanacademy/math-input": "^22.2.3",
-        "@khanacademy/perseus-core": "3.4.0",
-        "@khanacademy/perseus-linter": "^1.2.15",
-        "@khanacademy/pure-markdown": "^0.3.24",
-        "@khanacademy/simple-markdown": "^0.13.17",
-        "@khanacademy/wonder-blocks-banner": "4.0.5",
-        "@khanacademy/wonder-blocks-icon": "5.0.5",
-        "@khanacademy/wonder-blocks-icon-button": "6.0.5",
-        "@khanacademy/wonder-blocks-link": "7.0.5",
-        "@khanacademy/wonder-blocks-search-field": "4.0.6",
-        "@khanacademy/wonder-blocks-timing": "6.0.1",
-        "@khanacademy/wonder-blocks-tokens": "3.0.1",
-        "@khanacademy/wonder-blocks-toolbar": "5.0.5",
-        "@khanacademy/wonder-blocks-tooltip": "4.0.3",
-        "@khanacademy/wonder-stuff-core": "1.5.4",
-        "@phosphor-icons/core": "^2.0.2"
-    },
-    "devDependencies": {
-        "vite": "^5.1.0",
-        "@vitejs/plugin-react": "^4.3.3"
+        "@khanacademy/kas": "workspace:*",
+        "@khanacademy/kmath": "workspace:*",
+        "@khanacademy/math-input": "workspace:*",
+        "@khanacademy/perseus-core": "workspace:*",
+        "@khanacademy/perseus-linter": "workspace:*",
+        "@khanacademy/pure-markdown": "workspace:*",
+        "@khanacademy/simple-markdown": "workspace:*",
+        "@khanacademy/wonder-blocks-banner": "catalog:",
```

Comment:
`workspace:*` for in-repo dependencies, `catalog:` for the deps that we'll be provided at runtime (peer deps).

This should make managing peer and dev dependency versions so much easier!  Plus, Dependabot (Github) [just announced](https://github.blog/changelog/2025-02-04-dependabot-now-supports-pnpm-workspace-catalogs-ga/) full support for pnpm catalogs a few _days_ ago!